src/v2/api/controllers/UserController.ts

```diff
@@ -68,6 +69,44 @@ export class UserController {
     private studentMapper: StudentMapper,
   ) {}
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: UserResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidBodyException:
+      Username is not correct (a-zA-Z0-9_), or too short (min: 2), or too long (max: 40)
+      Username cannot be empty
+      Email is not an email
+      Email cannot be empty
+      State value is not in enum
+      State cannot be empty
+
+    AlreadyRegisteredException:
+      User is already registered`,
+  })
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiEndpoint({
+    summary: 'Create a new user by admin',
+    permissions: PERMISSION.ADMIN_USER_CREATE,
+  })
+  @Post('/createUser')
+  async createUser (@Body() body: UserByAdminDTO) {
+    return this.userMapper.getUser(
+      await this.userService.createUserByAdmin(body)
```

Comment:
create an intermediate variable for data that are returned by createUserByAdmin method and then pass it to getUser method